### ISSUE TITLE
Tooling and standardization for sporks.json

### ISSUE THREAD

**Issue by synzhu** [onflow/flow#814]
Context: https://github.com/onflow/flow/pull/809#issuecomment-1032943859

Once we establish some standard for the sporks.json format, we should have some tool to generate this file in order to avoid manual errors.

Also, I think it would be a good idea to create a new file per spork to retain the history of sporks instead of appending to the same file every time. This makes it easier to track down the details of a specific spork rather than looking through a large file. It's also likely that the sporks.json format will continue to change over time, and so it would be better to have separate files per spork rather than try to keep multiple formats in a single file.

**Comment by lkd70** [comment]
Agreed, a standard needs to be establish and some tooling needs putting in place to ensure the validity and formatting of the json.

Right now the sporks.json isn't valid JSON code as it contains a trailing curly bracket. #1186 
This renders it unusable by any JSON interpreter.
